IdentityCore/src/cache/token/MSIDCredentialCacheItem.m

```diff
@@ -206,6 +218,14 @@ - (NSDictionary *)jsonDictionary
     dictionary[MSID_ENROLLMENT_ID_CACHE_KEY] = _enrollmentId;
     dictionary[MSID_SPE_INFO_CACHE_KEY] = _additionalInfo[MSID_SPE_INFO_CACHE_KEY];
 
+    // Last Modification info (currently used on macOS only)
+    if (_lastModificationTime)
+    {
+        dictionary[MSID_LAST_MOD_TIME_CACHE_KEY] =
```

Comment:
use msidDateToTimestamp instead for consistency?

Reply:
I'd rather not since that truncates the fractional seconds. The added precision seems useful for debugging in this case.

Reply:
ok, makes sense. can you then add a different util or modify this one to accept precision as a parameter?

Reply:
ok, good idea

Reply:
moved this to NSDate+MSIDExtensions.m